connector/connectorRunner.go

```diff
@@ -55,15 +59,20 @@ func (cr *connectorRunner) Run() error {
 		return err
 	}
 
-	dataAggregator, err := process.NewDataAggregator(outportBlocksPool)
+	dataAggregator, err := process.NewDataAggregator(outportBlocksPool, converter)
+	if err != nil {
+		return err
+	}
+
+	s, writer, err := factory.CreateGRPCServer(cr.enableGrpcServer, cr.config.GRPC, outportBlocksPool, dataAggregator)
 	if err != nil {
 		return err
 	}
 
 	publisher, err := process.NewFirehosePublisher(
-		os.Stdout,
+		writer,
 		blockContainer,
-		protoMarshaller,
+		&process.ProtoMarshaller{},
```

Comment:
please add a separate variable for proto marshaller and rename `protoMarshaller` to `gogoProtoMarshaller` or something similar

Reply:
changed, but publisher is now in a factory.